pkg/storage/pod/annotations/generator_test.go

```diff
@@ -39,10 +39,12 @@ var _ = Describe("Annotations Generator", func() {
 		Expect(err).NotTo(HaveOccurred())
 
 		expectedAnnotations := map[string]string{
-			"pre.hook.backup.velero.io/container":  "compute",
+			"pre.hook.backup.velero.io/container": "compute",
+			//nolint: lll
```

Comment:
I prefer this is solved and not ignored. If you think there is a very (very!) good reason to do so, please convince us on the commit message.

Reply:
Done.